run.sh

```
@@ -1,19 +1,395 @@
+    export MARKETD_PORT
+}
+
+#  compose helpers 
+compose_up_services() {
+    # Start only the listed services (pass service names as args).
+    docker compose -f "$SCRIPT_DIR/docker-compose.yml" up -d --build "$@"
+}
+
+compose_down() {
+    docker compose -f "$SCRIPT_DIR/docker-compose.yml" down "$@"
+}
```

Shellcheck (0.11.0)

[warning] compose_down references arguments, but none are ever passed.

(SC2120)